core/src/main/java/org/elasticsearch/action/percolate/PercolateRequestBuilder.java

```diff
@@ -28,7 +28,8 @@
 import org.elasticsearch.common.xcontent.XContentType;
 import org.elasticsearch.index.query.QueryBuilder;
 import org.elasticsearch.search.aggregations.AggregatorBuilder;
-import org.elasticsearch.search.aggregations.pipeline.PipelineAggregatorBuilder;
+import org.elasticsearch.search.aggregations.PipelineAggregatorBuilder;
+import org.elasticsearch.search.aggregations.pipeline.AbstractPipelineAggregatorBuilder;
```

Comment:
Why do we need to import both `PipelineAggregatorBuilder` and `AbstractPipelineAggregatorBuilder` here?